native/cocos/application/BaseGame.h

```diff
@@ -56,13 +65,11 @@ class BaseGame : public CocosApplication {
         _windowInfo.flags  = _windowInfo.flags == -1 ? cc::ISystemWindow::CC_WINDOW_SHOWN |
                                                           cc::ISystemWindow::CC_WINDOW_RESIZABLE |
                                                           cc::ISystemWindow::CC_WINDOW_INPUT_FOCUS
-                                                    : _windowInfo.flags;
-        if (_windowInfo.x == -1 || _windowInfo.y == -1) {
+                                                     : _windowInfo.flags;
+        std::call_once(_windowIsCreated, [&]() {
             createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        } else {
-            createWindow(_windowInfo.title.c_str(),
-                         _windowInfo.x, _windowInfo.y, _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        }
+        });
```

Comment:
Need to call the interface to change the x, y coordinates of the window？

Reply:
no need, as x, y coordinates means actually nothing here..

Reply:
This is an interface that provides additional functionality. Although it is not currently called, developers may use it.
For example, we are currently displayed in the lower left corner. If the developer wants to change this operation, just set the coordinates of x and y directly.

Reply:
understand, still need to check x and y for the first init